### ISSUE TITLE
Deprecation warnings with Ember 3.27

### ISSUE THREAD

**Issue by boris-petrov** [jeffjewiss/ember-cli-postcss#760]
Using [global](https://github.com/jeffjewiss/ember-cli-postcss/blob/v7.0.0/index.js#L116) Ember has been deprecated in Ember 3.27 and will be removed in Ember 4.

It would be nice to fix this in both versions 6 and 7 of `ember-cli-postcss` as we are still using 6 (because some things are still incompatible with PostCSS 8).

**Comment by jeffjewiss** [comment]
Could you please try https://github.com/jeffjewiss/ember-cli-postcss/pull/763 and let me know if that works for PostCSS 8? If so I'll cut a patch version for V6 (postcss 7) as well. Thanks!

**Comment by boris-petrov** [comment]
@jeffjewiss - I believe that fixes the issue but the version of `ember-cli-postcss` isn't shown any more. Not that it matters so much. But wouldn't [such](https://github.com/emberjs/ember-inspector/blob/master/ember_debug/utils/ember.js) a fix be better?

**Comment by jeffjewiss** [comment]
Yeah, I'm open to adding a fix like the one you linked. Do you mind opening a PR?

**Comment by NullVoxPopuli** [comment]
With embroider, I got a hard error:

```
Uncaught ReferenceError: Ember is not defined
    <anonymous> Ember
register-version.js:1
```

![image](https://user-images.githubusercontent.com/199018/119162839-d7fbe580-ba28-11eb-8726-b3495140006e.png)

**Comment by boris-petrov** [comment]
@NullVoxPopuli - can you try the changes in the PR [I've opened](https://github.com/jeffjewiss/ember-cli-postcss/pull/764)?

**Comment by NullVoxPopuli** [comment]
your PR throws 
```
"Error: Could not find module `ember` imported from `(require)`
    at missingModule (http://localhost:4200/assets/vendor.js:256:11)
    at findModule (http://localhost:4200/assets/vendor.js:267:7)
    at requireModule (http://localhost:4200/assets/vendor.js:33:15)
    at http://localhost:4200/assets/vendor.js:2871:13"
    ```
    in the requireModule line

**Comment by boris-petrov** [comment]
I have no idea. That's the way they've fixed it in [ember-inspector](https://github.com/emberjs/ember-inspector/pull/1535).

**Comment by mydea** [comment]
IMHO the library stuff is not really useful anyhow nowadays, so I personally would not miss it if we simply removed this. I don't really see the benefit in adding some code to generate the library version vs. just looking at my package.json file to see which version I have installed :sweat_smile: But maybe others do use this functionality somehow, not sure. For me, ember-cli-postcss is not _such_ a critical addon that I would be especially interested in its version, compared to any other addon.

**Comment by jeffjewiss** [comment]
@NullVoxPopuli thanks for you help with this. I don't have an app with embroider available to test. Do you mind testing #763 with your app? I'm leaning towards just removing the version number as I'm not sure it provides much value. Thanks!

**Comment by NullVoxPopuli** [comment]
looks like it works!

**Comment by jeffjewiss** [comment]
#763 has now been merged, so this should be fixed by v7.0.2

Please reopen the issue if this continues to be a problem. Thanks!

**Comment by boris-petrov** [comment]
@jeffjewiss - thanks! Would you mind backporting this fix to 6.x?

**Comment by jeffjewiss** [comment]
@boris-petrov yeah of course. I've cherry-picked the commit from #763 and published v6.0.2. Could you please try that?

**Comment by boris-petrov** [comment]
@jeffjewiss - seems great, thank you for the quick fix!
